src/deprecated/ListGroupItem.js

```diff
@@ -0,0 +1,113 @@
+import React, { cloneElement } from 'react';
```

Comment:
This really should be showing `src/ListGroupItem.js` -> `src/deprecated/ListGroupItem.js` :panda_face: